ext/drreg/drreg.c

```diff
@@ -266,6 +497,88 @@ drreg_max_slots_used(OUT uint *max)
  * ANALYSIS AND CROSS-APP-INSTR
  */
 
+#ifdef SIMD_SUPPORTED
+
+static bool
+is_partial_simd_read(instr_t *instr, reg_id_t reg)
+{
+    int i;
+    opnd_t opnd;
+
+    for (i = 0; i < instr_num_srcs(instr); i++) {
+        opnd = instr_get_src(instr, i);
+        if (opnd_is_reg(opnd) && opnd_get_reg(opnd) == reg &&
+            opnd_get_size(opnd) < reg_get_size(reg))
```

Comment:
When comparing, use opnd_size_in_bytes(opnd_get_size(opnd)).

Reply:
Ah yes, I missed this. Thanks.